CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
 
   For Redirect uri:
 
   `http://localhost:5000/auth/myusa/callback`
 
-2. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
+2. Under `Select the API Scopes that your Application will use`: select `Email
+   Address`.
+
+3. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
```

Comment:
maybe we should:

1) tell peoplee they need to click "New API Key" next to the application they just created in myusa in order to create these keys
2) clarify what these are called in myusa? (Consumer Public Key, Consumer Secret Key)

this is perhaps overkill in terms of instructions but would be nice to have super explicit instructions. what do you think?